ginkgo.Jenkinsfile

```diff
@@ -63,7 +63,7 @@ pipeline {
             steps {
                 parallel(
                     "Runtime":{
-                        sh 'cd ${TESTDIR}; ginkgo --focus="RuntimeValidated*" -v -noColor'
+                        sh 'cd ${TESTDIR}; ginkgo --focus=" RuntimeValidated*" -v -noColor'
```

Comment:
Is this intentional?

Reply:
Yes! With this change will match only when start with Runtime, if not if you have DisabledRuntime will match ;-)